vector/src/main/java/im/vector/app/features/settings/notifications/VectorSettingsNotificationPreferenceFragment.kt

```diff
@@ -85,6 +85,21 @@ class VectorSettingsNotificationPreferenceFragment @Inject constructor(
             (pref as SwitchPreference).isChecked = areNotifEnabledAtAccountLevel
         }
 
+        findPreference<SwitchPreference>(VectorPreferences.SETTINGS_ENABLE_THIS_DEVICE_PREFERENCE_KEY)?.let {
+            it.setTransactionalSwitchChangeListener(lifecycleScope) { isChecked ->
+                if (isChecked) {
+                    FcmHelper.getFcmToken(requireContext())?.let {
+                        pushManager.registerPusherWithFcmKey(it)
```

Comment:
it's weird that in this case, we do not call `session.refreshPushers()` (I see this is not a change from this PR)